### PR TITLE
(Feature) Refine skip build logic to build relevant pull requests

### PR DESCRIPTION
**Pull request by eriese**
Each site checks pull requests for files changed in its own folder and in shared

### CODE REVIEW

File: sites/sanity/skip_build.mjs [hunk 1]
```diff
@@ -15,13 +15,32 @@ if (process.env.VERCEL_GIT_COMMIT_AUTHOR_LOGIN === 'dependabot[bot]') {
...
+// Always build develop branch
+if (branch === 'develop') {
   console.log('✅ - develop build - Proceed to build')
   process.exit(1)
 }
 
+// Only build pull requests that made changes to lab
+if (process.env.VERCEL_GIT_PULL_REQUEST_ID) {
+  try {
+    const changes = execSync(
```
Comment (joostdecock) [inline sites/sanity/skip_build.mjs:28]: We're missing the import
```js
import { execSync } from 'child_process'
```

**Review by joostdecock — APPROVED** [review]

**Comment by joostdecock** [comment]
LGTM. I've left some comments, but it's either cosmetics (comments copy-pasted without updating the site name) or things the linter is also yelling about.

So perhaps we can get there tests to pass and then merge.

File: sites/org/skip_build.mjs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import process from 'node:process'
+import { execSync } from 'child_process'
 
 const branches = ['develop', 'joost']
```
[the remark this answers is not in the document]
Comment (eriese) [reply sites/org/skip_build.mjs:4]: Line 4, you mean?

**Comment by eriese** [comment]
great. unfortunately even though you approved with the hope that I could just merge after changing, it considers that review stale so it wants a new one. womp womp

**Review by joostdecock — APPROVED** [review]

**Pull request merged**